src/lib.rs

```diff
@@ -311,6 +312,19 @@ pub use backtrace::Backtrace;
 
 mod quick_error;
 
+// This isn't a function to avoid getting an additional call frame.
+// Unfortunately that means we have to export it too.
+#[macro_export]
+macro_rules! error_chain_make_backtrace {
+    () => {
+        match ::std::env::var_os("RUST_BACKTRACE") {
+            Some(ref val) if val != "0" =>
+                Some(::std::sync::Arc::new($crate::Backtrace::new())),
+            _ => None
+        }
+    };
+}
```

Comment:
I wanted to avoid an unrelated additional frame in the backtrace. However, since (at least for me) there are already 3 such frames I guess it's not particularly problematic - I'll make it into a function.